docs/bugs.json

```
@@ -1,4 +1,15 @@
 [
+    {
+        "uid": "SOL-2022-2",
+        "name": "AbiEncodeFromCalldataNestedArraySizeValidationBug",
```

My naming suggestion from the chat:
```suggestion
        "name": "AbiEncodeNestedCalldataArraySizeValidation",
```

---

I went for the later suggestion of ``NestedCallataArrayAbiReencodingSizeValidation`` for now. Open to changing it further.